README.md

```diff
@@ -29,4 +29,14 @@ We love to receive feedback and input from the community, so if you found a pote
 ### Suggest improvements
 
 Do you think some things could be done better in the repo or have new ideas?
-**Open an issue** in the repo and share it in the `#dev-support channel`.
\ No newline at end of file
+**Open an issue** in the repo and share it in the `#dev-support channel`.
+
+## Bug Bounty Program
+
+SSV Network is committed to ensuring the security of our smart contracts. We've partnered with [Immunefi](https://immunefi.com/) to host a dedicated bug bounty program.
+
+If you believe you've identified a vulnerability within our project, we encourage you to report it via our [Immunefi Bug Bounty page](https://immunefi.com/bounty/ssvnetwork/). All submissions and communications regarding vulnerabilities will be managed by the Immunefi team.
```

Comment:
`If you believe you've identified a vulnerability within our project`
|I dont like the word `project` cause this assumes the node as well.  I would use instead `smart contract`

Reply:
good point!